verification/curator-service/ui/src/components/SourceTable.tsx

```diff
@@ -361,6 +366,29 @@ class SourceTable extends React.Component<Props, SourceTableState> {
                                     />
                                 ),
                             },
+                            {
+                                title: 'Notification recipients',
+                                field: 'notificationRecipients',
+                                tooltip:
+                                    'Email addresses of parties to be notified of critical changes',
+                                render: (rowData): string =>
+                                    rowData.notificationRecipients
+                                        ? rowData.notificationRecipients?.join(
+                                              '\n',
+                                          )
+                                        : '',
+                                editComponent: (props): JSX.Element => (
+                                    <ChipInput
+                                        defaultValue={props.value || []}
+                                        // material-ui-chip-input doesn't define this type.
+                                        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Can you get rid of this now that there isn't an `any` there?

Reply:
Done, thanks.